src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -391,10 +391,12 @@ export class CalcitePanel {
 
   renderContent(): VNode {
     return (
-      <section class={CSS.contentContainer} onScroll={this.panelScrollHandler} tabIndex={0}>
-        <slot />
+      <div class={CSS.contentWrapper}>
+        <section class={CSS.contentContainer} onScroll={this.panelScrollHandler} tabIndex={0}>
```

Comment:
@y0n4 can you verify that the tabIndex doesn't nee to be moved to the node that has overflow:auto?

Reply:
User is able to scroll using keyboard and navigate with tab keyboard. Moved those properties to the container that has overflow